### DISCUSSION TITLE
Is there a reason why .lyt files need to have absolute paths?

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#1326] [General]
I'm mixing and matching docker and local builds: docker is used to build the latest on our master and local builds to tinker with OpenROAD and ORFS.

The absolute paths in the .lyt files in the objects folder cause failures when starting with something built with docker and continuing with a local ORFS/OpenROAD.

Is there a reason why .lyt files have to use absolute paths?

There are couple of abspath references in ORFS Makefile.

**Comment by maliberty** [comment]
I don't know - try relative and see

**Reply by oharboe** [reply]
:crossed_fingers: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1327
